src/main/java/appeng/me/storage/VoidFluidInventory.java

```diff
@@ -90,4 +90,9 @@ public int getSlot() {
     public boolean validForPass(final int i) {
         return i == 2;
     }
+
+    @Override
+    public boolean getSticky() {
```

Comment:
If you extract it into separate interface like `HasSticky` with default implementation `() -> false` you can just add interface implementation without adding new method directly in each class

Reply:
To double check how this should be done do you want me to have this in an interface that is implemented by ONLY the classes that need it or should I leave it in IMEInventoryHandler.java and give it a default of false?